### PR TITLE
Random CI fiddling

### PR DESCRIPTION
**Pull request by jcape**
This PR is meant for random fiddling with CI. It may be noisy as I test various things out, but I'll mark as ready for review when it's ready.

### CODE REVIEW

**Comment by jcape** [comment]
Current dependencies on/for this PR:
* main
  * **PR #94** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/94" target="_blank">Graphite</a> 
    * **PR #95** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/95" target="_blank">Graphite</a> 
      * **PR #96** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/96" target="_blank">Graphite</a> 
        * **PR #97** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/97" target="_blank">Graphite</a> 
          * **PR #98** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/98" target="_blank">Graphite</a> 
            * **PR #99** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/99" target="_blank">Graphite</a> 
              * **PR #101** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/101" target="_blank">Graphite</a> 
              * **PR #100** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/100" target="_blank">Graphite</a> 
                * **PR #102** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/102" target="_blank">Graphite</a> 
                  * **PR #104** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/104" target="_blank">Graphite</a> 
                    * **PR #105** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/105" target="_blank">Graphite</a> 
                      * **PR #106** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/106" target="_blank">Graphite</a> 
                        * **PR #107** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/107" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/107?utm_source=stack-comment).

File: .markdownlint-cli2.jsonc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  // Disable some built-in rules
+  "config": {
+    "line-length": false
```
Comment (nick-mobilecoin) [inline .markdownlint-cli2.jsonc:4]: The main reason I looked into pulling markdownlint early into SGX is that the README.mds in the mobilecoin repo were inconsistent in regards to line lengths.  Some paragraphs were 200+characters long while some were wrapped at around 80.
  
I can understand if we feel that 80 is too narrow, but perhaps being more explicit on what we want is better than blanket disabling
```suggestion
    "line-length": {"line_length": 100}
```

FYI: @remoun as they had interest in markdownlint so might be interested in the outcome of this discussion.

**Review by nick-mobilecoin — APPROVED** [review]

**Review by awygle — APPROVED** [review]

File: .markdownlint-cli2.jsonc (hunk 1 above)
Comment (jcape) [reply .markdownlint-cli2.jsonc:4]: Right, my preference is "no limit"/"soft wrap" :-)

Comment (nick-mobilecoin) [reply .markdownlint-cli2.jsonc:4]: Doing some googling it seems that hard vs soft wrapping in markdown (and latex) files seems to be a religious debate with no clear winner.  My preference for hard wrapping is probably influenced by vim usage and easy reformatting via `gqq`.  Unfortunately not everyone understands that Vim is the best editor experience ever :P.  

Lacking a good argument that isn't preference, I have no more objections to disabling the line length check.

**Pull request merged**